terminal/terminal_other.go

```diff
@@ -0,0 +1,13 @@
+// +build !windows
```

Comment:
You are right, *_unix.go doesn't have any special meaning, but *_linux.go does. See [Build Constraints](https://golang.org/pkg/go/build/). Also, for examples within the delve code look in the /proc dir. 

Instead of ```terminal_unix.go```, it should be ```terminal_linux.go```.